src/Thenewboston/Bank/Models/BankTransaction.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using System;
+using Newtonsoft.Json;
+
 namespace Thenewboston.Bank.Models
```

Comment:
Looks like there's an `amount` field in the documentation that's missing here.

Reply:
The amount field should be there, no idea why it's not shown up because it's there on my fork, that's weird. No worries will figure that out